CMakeLists.txt

```diff
@@ -47,7 +47,9 @@ find_package(
 find_package(nlohmann_json REQUIRED)
 find_package(GEOS CONFIG REQUIRED)
 find_package(laslib CONFIG REQUIRED)
-find_package(libfork CONFIG REQUIRED)
+add_library(BS_thread_pool INTERFACE)
```

Comment:
these 3 lines don't do anything here since BS_thread_pool_SOURCE_DIR is not defined. I will remove them.

It still works since vcpkg puts the necessary header in the include path